.eslintrc.js

```diff
@@ -292,9 +294,19 @@ module.exports = {
     'prefer-object-spread/prefer-object-spread': 1,
   },
 
-  plugins: ['babel', 'import', 'flowtype', 'prefer-object-spread', 'cypress'],
+  plugins: defaultPlugins,
 
   overrides: [
+    // Cypress plugin, global, etc only for cypress directory
+    // https://github.com/cypress-io/eslint-plugin-cypress
+    // cypress clashes with jest expect()
+    {
+      files: ['**/cypress/**'],
+      plugins: [...defaultPlugins, 'cypress'],
```

Comment:
I think the existing plugins are still used; so the following should suffice:

```suggestion
      plugins: ['cypress'],
```


(I couldn't find this confirmed in the docs, but I just tested it on a separate project by introducing an error that needed the `graphql` plugin to work, and it still worked after this change.)

Reply:
that would be simpler... it makes sense, since for example presets also provide plugins that are loaded

Reply:
With this we could go back to not using variables I think too :raised_hands:

Reply:
yep thats what I did! it was nice to elevate the plugins list though, it's hard to see what plugins are enabled at first glance